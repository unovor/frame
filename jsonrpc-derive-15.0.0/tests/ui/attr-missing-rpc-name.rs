use jsonrpc_derive::rpc;

#[rpc]
pub trait Rpc {
	/// Returns a protocol version
	#[rpc]
	fn protocol_version(&self) -> Result<String>;
}

fn main() {}
