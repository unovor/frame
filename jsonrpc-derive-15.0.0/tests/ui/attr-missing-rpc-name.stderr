error: rpc attribute should have a name e.g. `name = "method_name"`
 --> $DIR/attr-missing-rpc-name.rs:5:2
  |
5 |       /// Returns a protocol version
  |  _____^
6 | |     #[rpc]
7 | |     fn protocol_version(&self) -> Result<String>;
  | |_________________________________________________^
